node/src/actors/chain_manager/handlers.rs

```diff
@@ -289,7 +289,7 @@ impl Handler<AddBlocks> for ChainManager {
                     }
                 }
```

Comment:
We should add debug logs to detect unhandled `AddBlocks` messages. One here and one in `StateMachine::Synced => {}`.

```suggestion
                } else {
                    log::debug!("Unhandled AddBlocks message");
                }
```